frontend/src/styles/home/FeaturedProjectCard.module.css

```diff
@@ -1,33 +1,19 @@
-
 #cardContainer {

```

Comment:
throughout: there are many `text-align: center` styling rules for various elements (via id)-- why didn't you just have one rule to style the justification of the text via the container?

Reply:
I actually tried that but I _think_ it was messing up the desktop level styling so i got rid of it